packages/ui-components/src/components/Badge/Badge.types.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { KnownIcons } from "../Icon/Icon.component"
+
+// To Do: Externalise? Similar is used in Message component
```

Comment:
No, we should not do it please check this article https://sandimetz.com/blog/2016/1/20/the-wrong-abstraction#:~:text=Once%20an%20abstraction%20is%20proved%20wrong%20the%20best,abandon%20the%20wrong%20abstraction%20sooner%20rather%20than%20later.